examples/mssql-public/main.tf

```
@@ -19,6 +19,7 @@ provider "google-beta" {
   region  = var.region
 }
 
+# [START cloud_sql_sqlserver_instance_create_public]
 module "mssql" {
   source               = "../../modules/mssql"
```

For examples included in the docs, we should source from the registry:

```suggestion
  source  = "GoogleCloudPlatform/sql-db/google//modules/mssql"
  version = "~> 5.0"
```

---

changes are on the way

---

Thanks, please be sure to do the same for all the examples.

---

ok, thanks.